client/lobby/SelectionTab.h

```diff
@@ -11,6 +11,7 @@
 
 #include "CSelectionBase.h"
 #include "../../lib/mapping/CMapInfo.h"
+#include "../widgets/Images.h"
```

Comment:
Prefer forward declarations to includes in headers
```suggestion
class CPicture;
```